Spike/refactor translation mp service

IngageMaddoxT opened the pull request.

IngageMaddoxT commented:
@danrye Please review pull request.  I combined the Update, Create, Delete Page Services into a single file.  All using the Platform Util now to remove redundant code.  Properly updated namespaces.

Have a good weekend.

danrye commented on MinistryPlatform.Translation.Test/TranslationTests.cs:
```diff
@@ -174,7 +174,7 @@ public void ShouldNotDeleteResponse()
 
             //Try to delete Response, should fail
-            Assert.Throws<FaultException<ExceptionDetail>>(() => DeletePageRecordService.DeleteRecord(382, responseId, null, token));
+            Assert.Throws<FaultException<ExceptionDetail>>(() => MinistryPlatformService.DeleteRecord(382, responseId, null, token));
```
We should update 382 to use value from ConfigurationManager.  Not really part of this PR, I know.

danrye commented on crds-angular/Services/AccountService.cs:
```diff
@@ -74,7 +74,7 @@ public AccountInfo getAccountInfo(string token)
             householdDictionary["Household_Source_ID"] = Convert.ToInt32(ConfigurationManager.AppSettings["Household_Default_Source_ID"]);
 
-            recordId = MinistryPlatform.Translation.Services.CreatePageRecordService.CreateRecord(Convert.ToInt32(ConfigurationManager.AppSettings["Households"]), householdDictionary, token);
+            recordId = MinistryPlatform.Translation.Services.MinistryPlatformService.CreateRecord(Convert.ToInt32(ConfigurationManager.AppSettings["Households"]), householdDictionary, token);
```
Should be able to remove MinistryPlatform. Translation.Services from each of these blocks.  Again not part of this PR, but while we're here.

danrye commented on MinistryPlatform.Translation/Services/AuthenticationService.cs:
```diff
@@ -52,7 +51,7 @@ public static Boolean ChangePassword(string token, string newPassword)
             {
                 var record = GetPageRecordService.GetRecordsDict(Convert.ToInt32(ConfigurationManager.AppSettings["ChangePassword"]), token).Single();
```
Shouldn't we remove GetPageRecordService also? @IngageMaddoxT

danrye commented:
@IngageMaddoxT I made some changes, please review.

IngageMaddoxT commented:
@danrye I reviewed things.  Looks like you addressed all of your concerns.  I cleaned up the Lookup service a bit and merged dev back into the branch.  Please review again.  Thanks for your help.
